docs/backend_architecture/content/api_endpoints.rst

```diff
@@ -1,6 +1,26 @@
 API endpoints
 -------------
 
+URL Namespacing
+~~~~~~~~~~~~~~~
+
+Kolibri uses a consistent URL namespacing pattern throughout the codebase. All URLs follow the format:
+
+    >>> kolibri:<app_name>:<resource_name>_<endpoint>
```

Comment:
This isn't quite true for "core" URLs, as they all use "core" rather than their specific app_name.

It might be better to call this `plugin_module` with a note of a special exception for `core` which is not in a plugin module.